nrfx/hal/nrf_ccm.h

```diff
@@ -337,12 +317,11 @@ typedef enum
     NRF_CCM_TASK_KSGEN        = offsetof(NRF_CCM_Type, TASKS_KSGEN),        ///< Start generation of key-stream.
 #endif
 #if NRF_CCM_HAS_TASK_CRYPT
-    NRF_CCM_TASK_CRYPT        = offsetof(NRF_CCM_Type, TASKS_CRYPT),        ///< Start encryption/decryption.
-#endif
-    NRF_CCM_TASK_STOP         = offsetof(NRF_CCM_Type, TASKS_STOP),         ///< Stop encryption/decryption.
-#if NRF_CCM_HAS_TASK_START
+    NRF_CCM_TASK_START        = offsetof(NRF_CCM_Type, TASKS_CRYPT),        ///< Start encryption/decryption.
```

Comment:
@nika-nordic is this enum rename of a bug?
NRF_CCM_TASK_CRYPT => NRF_CCM_TASK_START (#if NRF_CCM_HAS_TASK_CRYPT)

Reply:
This is a cleanup. TASKS_START and TASKS_CRYPT are the same thing, only renamed in IP revisions. HAL was aligned to hide this HW diff. Same story with EVENT_END and EVEND_ENDCRYPT.

Reply:
The problem is that this is used by users, so if it is just renamed the code using it does not build anymore :(

Reply:
Right, users have to align to new names when switching to 3.7. We do not guarantee API compatibility between minor versions for HALs, only for drivers.

Reply:
( CC @cvinayak )
Are you guys sure this is a good idea? (and if so, shouldn't it have been mentioned in the changelog?)

Reply:
discussed offline with @aescolar to align bsim models to nrf_ccm changes